content/post/projects/flutter-bond/index.ar.md

```diff
@@ -0,0 +1,42 @@
+---
+title: فلاتر بوند
+description: إطار عمل لتطوير تطبيقات الهاتف المحمول مبني على فلاتر
+slug: flutter-bond
+date: 2024-08-27 21:00:00+0000
+image: flutter-bond.png
+categories:
+  - Project
+  - Software
+languages:
+  - English
+tags:
+  - Flutter
+  - Dart
+  - Mobile Development
+  - Open Source
+  - Code
+  - Libraries
+  - Programming
+---
+
+[فلاتر بوند](https://github.com/onestudio-co/flutter-bond) هو إطار عمل لتطوير تطبيقات الهاتف المحمول مبني على فلاتر، تم تطويره بفخر في غزة من قبل مهندس فلسطيني متحمس. مستوحى من مبادئ الهندسة المعمارية النظيفة ويهدف إلى تبسيط التطوير، يوفر ميزات مثل:
+
+* إدارة حالة التطبيق بسلاسة.
+* إجراء طلبات API مبسطة باستخدام Bond Fire.
+* التنقل والتوجيه السهل بين الشاشات.
+* بنية قابلة للتوسيع (Bond Core).
+* دعم واجهة سطر الأوامر (Bond CLI) لتطوير أسرع.
+
+من خلال التركيز على تقليل التعليمات البرمجية المكررة، يهدف Flutter Bond إلى جعل تطوير تطبيقات الهاتف المحمول أكثر متعة وكفاءة.
+
+**تعلم فلاتر بوند**
```

Comment:
To keep the website simple and avoid stale posts, we avoid adding details like this and leave it for the user to discover via the project URL.

So you can remove it from this line and afterward.

Reply:
Ok, I will do that